RUFAS/biophysical/animal/bedding/bedding.py

```diff
@@ -0,0 +1,146 @@
+from RUFAS.biophysical.animal.data_types.bedding_types import BeddingType
+
+
+class Bedding:
+    """
+    Abstract base class for all bedding types.
+
+    This class provides a base for all bedding types. It initializes with a configuration of bedding
+    attributes and includes methods for calculating various bedding properties.
+
+    Attributes
+    ----------
+    name : str
+        The name of the bedding.
```

Comment:
```suggestion
        Unique identifier to reference this bedding configuration.
```
Just pulling these from the updated metadata for consistency